payments-ui-core/src/main/java/com/stripe/android/paymentsheet/forms/PaymentMethodRequirements.kt

```diff
@@ -1,27 +1,36 @@
 package com.stripe.android.paymentsheet.forms
 
 import android.os.Parcelable
+import androidx.annotation.RestrictTo
 import kotlinx.parcelize.Parcelize
 
```

Comment:
Needed to be exposed so that the SupportedPaymentMethodKtx in Paymentsheet can use the PaymentConfiguration and the Requirements.  The requirements needed to be moved to payments-core-ui so that the data specs could be put in teh LpmRepository.